Sources/SmithyTestUtil/XMLComparator.swift

```diff
@@ -29,10 +29,11 @@ public struct XMLComparator {
 }
 
 private struct XMLElement: Hashable {
```

Comment:
This type is used for checking the equivalence of two pieces of XML.

I have modified it to be aware of XML namespaces (it was not before).